survey-sdk/src/test/java/co/nimblehq/surveysdk/serializer/DataTest.kt

```diff
@@ -0,0 +1,243 @@
+package co.nimblehq.surveysdk.serializer
+
+const val JSON_SURVEY_DETAIL_TEST: String = "{\n" +
```

Comment:
@luongvo noted for your recommendation. With large data, we should put them as json file.
In this case, I use resource string as the data is small and we should not put more effort to read and parse json file.

Reply:
@doandinh yup, but be aware that json format is better in this case to define the response in raw format as from the server-side. I can say, it would be nicer to have 🤓